data/Sri Dasam Granth/1390.json

```diff
@@ -109,7 +109,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "",
+              "translation": "Whosoever puts faith on the oath of Quran,",
```

Comment:
Remove comma?

Reply:
This is not 1 full sentence, rather 1 sentence split over 2 lines.

Reply:
This is not 1 full sentence, rather 1 sentence split over 2 lines.